### PR TITLE
Update default_extension_map.dart

### PR DESCRIPTION
**Pull request by zi6xuan**
Append the heic and heif file format.

### CODE REVIEW

**Comment by zi6xuan** [comment]
@googlebot I signed it!

**Comment by bhayatus** [comment]
Would be nice if this was merged, right now I have an annoying workaround to account for heic files not being recognized when using `lookupMimeType`.

**Comment by jadenwjh** [comment]
Hi @jakemac53, will this be an upcoming feature? I think this addition has been requested multiple times, please have a look :D

**Comment by kevmoo** [comment]
We should have tests and a changelog update...please! 😄

File: lib/src/default_extension_map.dart [hunk 1]
```diff
@@ -993,4 +993,6 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'zir': 'application/vnd.zul',
   'zirz': 'application/vnd.zul',
   'zmm': 'application/vnd.handheld-entertainment+xml',
+  'heic': 'image/heic',
```
Comment (jakemac53) [inline lib/src/default_extension_map.dart:996]: sort these alphabetically please so its easier to find what is supported

**Review by natebosch — APPROVED** [review]

**Pull request merged**